src/Workspaces/Core/Portable/Shared/Extensions/SourceTextExtensions.cs

```diff
@@ -17,9 +19,12 @@ namespace Microsoft.CodeAnalysis.Shared.Extensions;
 
 internal static partial class SourceTextExtensions
 {
-    // char pooled memory : 8K * 256 = 2MB
+    // char array length: 4k characters. 4K * 256 * 2 (bytes per char) = 4MB
```

Comment:
>4MB [](http://example.com/codeflow?start=73&length=3)

nit: isn't the original 2MB size correct here?

Reply:
yes.  will fix after test run happens.  i want to see if anything breaks.

Reply:
Done.